boule/ellipsoid.py

```diff
@@ -419,7 +419,7 @@ def _spherical_to_geodetic_terms(self, spherical_latitude, radius):
         big_d = k * radius * cos_latitude / (k + self.first_eccentricity ** 2)
         return k, big_z, big_d
 
-    def normal_gravity(self, latitude, height):
+    def normal_gravity(self, latitude, height, si_units=False):
```

Comment:
```suggestion
    def normal_gravity(self, latitude, height, si_units=False): # pylint: disable=too-many-locals
```

Reply:
This should make pylint stop complaining about too many variables. In this case, there is no way around it. You might need to run black again after this if the line is too long.